springboot/src/main/java/com/codecool/terraformingmarsforum/service/LeaguePostService.java

```
@@ -36,4 +38,9 @@ private LeaguePostModel createLeaguePostModel(LeaguePost leaguePost) {
                 .numberOfRounds(leagueDetail.getNumberOfRounds())
                 .build();
     }
+
+    public void addCommentToPostByPostId(Long postId, Comment comment) {
+        Optional<LeaguePost> post = leaguePostRepository.findById(postId);
+        post.ifPresent(leaguePost -> leaguePost.getComments().add(comment));
+    }
```

Here we need to throw an exception if post is not found